plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/SourceMeterReader.java

```diff
@@ -81,16 +81,8 @@ private TestCaseResult parseSourceMeterItem(String vuln, String file) throws Exc
         tcr.setCategory(vuln);
         tcr.setEvidence(file);
         tcr.setCWE(cweLookup(vuln));
-
-        String testno = file.substring(file.length() - 5); // extract test number
-        try {
-            tcr.setNumber(Integer.parseInt(testno));
-            return tcr;
-        } catch (NumberFormatException e) {
-            System.out.println("> Parse error " + file + ":: " + testno);
-        }
-
-        return null;
+        tcr.setNumber(testNumber(file));
```

Comment:
Again, if the finding is NOT in a test case, we don't want to return the new TCR.

Reply:
As the previous comment